zeppelin-server/src/main/java/org/apache/zeppelin/realm/LdapRealm.java

```diff
@@ -329,8 +329,8 @@ private boolean hasAllowedAuthenticationRules(PrincipalCollection principals,
       systemLdapCtx = ldapContextFactory.getSystemLdapContext();
       return rolesFor(principals, username, systemLdapCtx,
         ldapContextFactory, SecurityUtils.getSubject().getSession());
-    } catch (AuthenticationException ae) {
-      ae.printStackTrace();
+    } catch (Throwable t) {
+      t.printStackTrace();
```

Comment:
Use `Logger.error` to logging error message

Reply:
@zjffdu i wondered if there was a reason it was printing the stack trace.   log.error makes more sense.  i'll change it

Reply:
@zjffdu both changes made.  I used log warn instead of error as this message happens in non-error scenarios.   Anytime you have 2 realms configured this warning will throw for one of the 2 realms as expected.